.travis.yml

```diff
@@ -52,9 +106,64 @@ script:
     - git checkout -b travis-builds binaries/travis-builds
     - git config --global user.email "builds@travis-ci.com"
     - git config --global user.name "Travis CI"
-    - cp ${GOPATH}/bin/skydive skydive-latest
-    - git add skydive-latest
+    - cp ${GOPATH}/bin/skydive skydive-${TRAVIS_OS_NAME}
+    - git add skydive-${TRAVIS_OS_NAME}
     - git commit -m "${BUILD_TAG} travis build" --amend
     - git config credential.helper "store --file=.git/credentials"
     - echo "https://${GITHUB_USERNAME}:${GITHUB_TOKEN}@github.com" > .git/credentials
     - git push -f -q binaries travis-builds
+
+before_deploy:
+    - git config --global user.email "builds@travis-ci.com"
+    - git config --global user.name "Travis CI"
+    - make DESTDIR=$HOME SKYDIVE_PKG=skydive-${TRAVIS_OS_NAME}-${TRAVIS_TAG} localdist
```

Comment:
localdist target didn't exist anymore on master, please use make dist, and rebase